src/App.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
+import BooksList from './components/BooksList';
+import BookForm from './components/BookForm';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Hello</h1>
-    </div>
-  );
-}
+const App = () => (
+  <div>
```

Comment:
- Currently the`div` will create a node which we don't need, but we are had to create because the react convention. In such cases use [Fragment](https://www.barrymichaeldoyle.com/fragment/)